src/media-query-provider.js

```diff
@@ -11,14 +11,12 @@ const defaultQueries = {
 
 class MediaQueryProvider extends React.Component {
 
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
 
     this.state = {
-      media: {},
+      media: this.queryMedia(props),
```

Comment:
what happens when this gets run in node

Reply:
uses this lib internally: https://github.com/ericf/css-mediaquery/blob/master/index.js
When values are passed it doesnt rely on `window`